### PR TITLE
Update build

### PR DESCRIPTION
**Pull request by SuperQ**
* Remove obsolete release tool.

### CODE REVIEW

File: consul_exporter.go [hunk 1]
```diff
@@ -397,6 +396,7 @@ func main() {
 		log.Fatalln(err)
 	}
 
+	//lint:ignore SA1019 relax staticcheck verification.
```
Comment (beorn7) [inline consul_exporter.go]: This can go now. (Just resolve the merge conflict to see why.)

Comment (SuperQ) [reply consul_exporter.go]: Yup, thanks, done.

**Comment by SuperQ** [comment]
Tests are now failing due to upstream issue with golang linter.

**Comment by simonpasquier** [comment]
I've updated Makefile.common to include https://github.com/prometheus/prometheus/pull/5658

**Pull request merged**